contract/vm.go

```diff
@@ -791,6 +731,37 @@ func (ce *executor) gas() uint64 {
 	return uint64(C.lua_gasget(ce.L))
 }
 
+func (ce *executor) vmLoadCode(id []byte) {
+	var chunkId *C.char
+	if ce.ctx.blockInfo.ForkVersion >= 3 {
+		chunkId = C.CString("@" + types.EncodeAddress(id))
+	} else {
+		chunkId = C.CString(hex.Encode(id))
+	}
+	defer C.free(unsafe.Pointer(chunkId))
+	if cErrMsg := C.vm_loadbuff(
+		ce.L,
+		(*C.char)(unsafe.Pointer(&ce.code[0])),
+		C.size_t(len(ce.code)),
+		chunkId,
+		ce.ctx.service-C.int(maxContext),
+	); cErrMsg != nil {
+		errMsg := C.GoString(cErrMsg)
+		ce.err = errors.New(errMsg)
+		ctrLgr.Debug().Err(ce.err).Str("contract", types.EncodeAddress(id)).Msg("failed to load code")
+	}
+}
+
+func (ce *executor) vmLoadCall() {
+	if cErrMsg := C.vm_loadcall(
+		ce.L,
+	); cErrMsg != nil {
```

Comment:
On this case the argument can be on the same line as the function

Reply:
> Why not separate the call from the `if` clause? The same for the function above

it can limiting error within if scope. ( and also shorten line. )